backend/src/test/java/dataentities/Pet.java

```diff
@@ -0,0 +1,39 @@
+package dataentities;
+
+import io.qameta.allure.internal.shadowed.jackson.annotation.JsonProperty;
+import io.qameta.allure.internal.shadowed.jackson.annotation.JsonPropertyOrder;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@JsonPropertyOrder({"name", "photoUrls"})
```

Comment:
i guess you don't need this annotation

Reply:
Resolved. But need to investigate more on the topic...